### DISCUSSION TITLE
Not working RPCs and localhost is prohibited

### DISCUSSION THREAD

**Discussion by dappbuildernet** [WalletConnect/walletconnect-monorepo#3971] [Developer Support]
Hello,

Just started an hour ago:

1. RPCs are not working - can't fetch any data on all of my dapps.
2. Localhost is now blocked "because an ancestor violates...." though localhost is supposed to be allowed by default...
3. Even without whitelisting of links still getting 2nd error on any other link.

![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/145851892/7087af5b-008a-4679-bcdd-34cca0deafd7)

**Comment by glitch-txs** [comment]
hi, are you still experiencing this issue?

**Comment by dappbuildernet** [comment]
Hi,

Thank you for following up!

1. For the RPCs - sometimes, mostly often with Base chain, not enough often to be a really huge issue (at least at the current stage of development).
2. For the Content Security Policy - still getting console.log errors on localhost but everything is still working so not an issue.
![Image 27-11-2023 at 15 06](https://github.com/WalletConnect/walletconnect-monorepo/assets/145851892/651bdd51-a83f-4bba-8551-70b80036b716)

Have a wonderful day!

**Reply by glitch-txs** [reply]
Which library are you using and version?

**Reply by dappbuildernet** [reply]
I have installed the original recommended config from the web3modal docs with 3.3.1 wagmi/react and later also updated all the packages to the latest a couple of days ago

Everything else is working fine, that console message does not really change the behavior of the dapp on the localhost so it is just a message... the dapp is still working. 

BTW right now there is another ongoing failure of Base chain RPC...
POST https://rpc.walletconnect.com/v1/?chainId=eip155:8453&projectId=............. 400 (Bad Request)

**Reply by glitch-txs** [reply]
can you show your wagmi version

**Reply by glitch-txs** [reply]
also can you share your website or code? and project ID if possible

**Reply by 97albertus** [reply]
i wanna chime in on this error, in my case the cause was that the Adblock extension was blocking the frame
